src/main/java/uk/gov/digital/ho/hocs/info/api/dto/FieldDto.java

```diff
@@ -55,4 +60,33 @@ public static FieldDto from(final Field field) {
                 field.isActive(),
                 childField);
     }
+
+    public static FieldDto fromWithDecoratedProps(final Field field) {
+        final FieldDto childField = field.getChild() != null ? FieldDto.from(field.getChild()) : null;
+
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
It's not a show stopper for this, so up to you if you want to change it. `ObjectMapper` is a bit of a heavyweight object. Since this is a method called within a stream we're instantiating an instance with each iteration in the stream, it might be wiser and better performing to process the Field to FieldDto's within the SchemaService, passing an autowired instance of `ObjectMapper` to the `fromWithDecoratedProps` method from the service, that way it won't be instantiated with each loop in the stream.